src/inspector_agent.h

```diff
@@ -50,39 +51,53 @@ class Agent {
   explicit Agent(node::Environment* env);
   ~Agent();
 
+  // Create client_, may create io_ if option enabled
   bool Start(v8::Platform* platform, const char* path,
              const DebugOptions& options);
+  // Stop and destroy io_
   void Stop();
 
-  bool IsStarted();
+  bool IsStarted() { return !!client_; }
+
+  // IO thread started, and client connected
   bool IsConnected();
+
+
   void WaitForDisconnect();
   void FatalException(v8::Local<v8::Value> error,
                       v8::Local<v8::Message> message);
+
+  // These three APIs called by the WS protocol and JS binding to create
```

Comment:
Is this a typo/supposed to be `are called by`? (Also, I’d just say `methods` instead of `APIs` – at least to me a single API is usually a set of methods or similar things)